ccdaservice/oe-blue-button-generate/lib/sectionLevel2.js

```diff
@@ -365,6 +365,9 @@ exports.planOfCareSection = function (htmlHeader, na) {
             key: "section",
             attributes: condition.isNullFlavorSection('plan_of_care'),
             content: [
+                // @see http://www.hl7.org/ccdasearch/templates/2.16.840.1.113883.10.20.22.2.10.html
+                // They keep renaming this section, but ccda calls this Plan of Treatment Section (V2)
+                fieldLevel.templateIdExt("2.16.840.1.113883.10.20.22.2.10", "2014-06-09"),
```

Comment:
Thought I got all the rel2.1 headers. We use assessment and a plan of care and not the combined.